demos/jans-tent/.github/workflows/flake8-lint.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
```

Comment:
@moabu this is linting

Reply:
We can include this in general for all python code. I'll take care of this one.